Corona-Warn-App/src/main/res/values-pl/antigen_strings.xml

```diff
@@ -4,201 +4,205 @@
+    <string name="ag_homescreen_card_rapidtest_status_outdated_test">"Wstecz"</string>
     <!-- XTXT: rapid test homescreen card: result positive - check results with PCR test -->
-    <string name="ag_homescreen_card_rapidtest_body_result_positive_pcr_check">"Wykonaj test PCR, aby zweryfikować wynik tego testu."</string>
+    <string name="ag_homescreen_card_rapidtest_body_result_positive_pcr_check">"Niepoprawny kod QR"</string>
 
     <!-- Body -->
     <!-- XTXT: rapid test homescreen card: body - no result -->
-    <string name="ag_homescreen_card_rapidtest_body_no_result">"Ustalanie wyniku Twojego szybkiego testu jeszcze się nie zakończyło."</string>
+    <string name="ag_homescreen_card_rapidtest_body_no_result">"Kod QR jest niepoprawny lub został już zarejestrowany na innym smartfonie. Otrzymasz swój wynik testu z ośrodka wykonującego testy lub laboratorium niezależnie od ważności kodu QR. W przypadku zdiagnozowania u koronawirusa otrzymasz powiadomienie z organu ds. zdrowia publicznego."</string>
     <!-- XTXT: rapid test homescreen card: body - outdated test -->
-    <string name="ag_homescreen_card_rapidtest_body_outdated_test">"Twój szybki test ma więcej niż 48 godzin i nie będzie już tutaj wyświetlany."</string>
+    <string name="ag_homescreen_card_rapidtest_body_outdated_test">"TAN jest nieprawidłowy."</string>
     <!-- XTXT: homescreen card: body - negative -->
-    <string name="ag_homescreen_card_rapid_body_result_date">"Wykonany dnia %1$s"</string>
+    <string name="ag_homescreen_card_rapid_body_result_date">"TAN jest nieprawidłowy lub został już użyty. Więcej informacji można uzyskać dzwoniąc na numer wymieniony w sekcji „Poproś o TAN”."</string>
 
     <!-- ####################################
         Homescreen cards - PCR
     ###################################### -->
     <!-- XHED: PCR homescreen card: title -->
-    <string name="ag_homescreen_card_pcr_title">"Test PCR"</string>
+    <string name="ag_homescreen_card_pcr_title">"Wstecz"</string>
     <!-- XBUT: PCR homescreen card: clear test button -->
-    <string name="ag_homescreen_card_pcr_clear_test_button">"USUŃ TEST"</string>
+    <string name="ag_homescreen_card_pcr_clear_test_button">"Błąd"</string>
 
     <!-- Body -->
     <!-- XTXT: PCR homescreen card: body - no result -->
-    <string name="ag_homescreen_card_pcr_body_no_result">"Ustalanie wyniku Twojego testu PCR jeszcze się nie zakończyło."</string>
+    <string name="ag_homescreen_card_pcr_body_no_result">"Nie można było wygenerować numeru TAN do przesłania. Skontaktuj się z techniczną infolinią, wybierając opcje: Informacje o aplikacji -&gt; Infolinia techniczna."</string>
     <!-- XTXT: homescreen card: body - negative -->
-    <string name="ag_homescreen_card_pcr_body_result_date">"Test zarejestrowany dnia %1$s"</string>
+    <string name="ag_homescreen_card_pcr_body_result_date">"Wstecz"</string>
```

Comment:
This string has wrong format

Reply:
something went wrong as the whole polish file was changed even if we didn't changed that much, or translation expert will take care of this.